verification/curator-service/ui/src/components/App/App.tsx

```diff
@@ -571,15 +571,24 @@ export default function App(): JSX.Element {
                         {user && (
                             <>
                                 <Typography>
-                                    <a
+                                {process.env.REACT_ENV_TYPE === "staging" ? 
+                                <a
                                         className={classes.mapLink}
                                         data-testid="mapLink"
-                                        href="https://map.covid-19.global.health/"
+                                        href="http://dev-map.covid-19.global.health/"
```

Comment:
I would suggest to use ternary operator in href directly instead of conditionally rendering two `<a>` components that differ only in href